src/CMakeLists.txt

```diff
@@ -737,6 +721,23 @@ if(WIN32)
   # The UNICODE and _UNICODE definitions are needed to make
   # (QString) mudlet::getShortPathName(const QString& name) work in Windows:
   target_compile_definitions(${EXE_MUDLET_TARGET} PRIVATE UNICODE _UNICODE)
+
+  # Select Windows executable icon based on build type
+  if(APP_BUILD MATCHES "-ptb.+")
```

Comment:
This will need to respect the `WITH_VARIABLE_SPLASH_SCREEN` (which I am going to change to `WITH_BUILD_BRANDING`) so that those who want/need to force Mudlet to appear as the "Normal" thing even when `MUDLET_VERSION_BUILD` has been set to something...